gen_cert.py

```diff
@@ -1878,10 +1886,11 @@ def fontlist_with_style(a_style):
 
         # SECTION: Extra achievements
         achievements_string = ""
-        achievements_description_string = self.interstitial_texts[grade]
+        achievements_description_string = self.interstitial_texts[grade].decode('utf-8')
```

Comment:
Did this previously break without the `decode` bit? It doesn't look like the dict is actually changing, so I'm curious about the necessity of this.